services/pkgreview/package-rating.service.js

```diff
@@ -0,0 +1,80 @@
+'use strict'
+
+const Joi = require('@hapi/joi')
+const { BaseJsonService } = require('..')
+
+const schema = Joi.object({
+  name: Joi.string().required(),
+  type: Joi.string().required(),
+  icon: Joi.string().required(),
+  starString: [Joi.string().optional(), Joi.allow(null)],
+  reviewsCount: Joi.number(),
+  rating: [Joi.number().optional(), Joi.allow(null)],
```

Comment:
Based on the info you shared in the original issue, this seems like it will always be a decimal number between 0 and 5, so let's reflect that in the schema:

```suggestion
  rating: Joi.number().min(0).max(5).required(),
```

Reply:
Doing it!

Reply:
When the package isn't reviewed by anyone, there wouldn't be any rating, so `rating: null` in such case.